### ISSUE TITLE
Unicode characters upset vertical space

### ISSUE THREAD

**Issue by coljac** [yudai/gotty#150]
The terminal emulator Gotty uses appears to have problems with some Unicode characters taking up extra vertical space.

To reproduce: 
```
gotty echo '                                                                                                                     🐍 Python 3.5.2 
OK  |
ᚖᚖ  |   
OK  |'
```

Expected behaviour: The pipe symbols all line up as they do in a regular terminal with a monospace font.
Actual behaviour: These characters take up extra vertical space and push everything to the right of them out.

![vertical_space](https://user-images.githubusercontent.com/191407/27942815-75187ca0-631d-11e7-8575-c8762c0f2df0.png)

This appears to be true for github markdown too - so I imagine this is hard to fix.

**Comment by yudai** [comment]
With the current master code that uses xtermjs, Safari and Firefox on OSX can show the characters with the proper alignment.

Firefox:
<img width="71" alt="screen shot 2017-08-21 at 6 56 13 pm" src="https://user-images.githubusercontent.com/33192/29545355-aeedebba-86a2-11e7-91c2-db4832485bed.png">

Safari:
<img width="81" alt="screen shot 2017-08-21 at 6 56 34 pm" src="https://user-images.githubusercontent.com/33192/29545351-a38ed504-86a2-11e7-9e63-68300423ae18.png">

Chrome: 
<img width="59" alt="screen shot 2017-08-21 at 6 56 50 pm" src="https://user-images.githubusercontent.com/33192/29545335-8c774ffe-86a2-11e7-9afb-e5b8fa35e6df.png">

However, Chrome somehow cannot show the character itself.  I assume this is a Chrome issue that the browser cannot properly handle fallbacks for missing unicode fonts.

**Comment by yudai** [comment]
I found a solution and fixed in 4d682aa01da4236574046e4cb1a6cd11aefa05a5 .

**Comment by coljac** [comment]
Can confirm this looks good now (though upgrading to `master` breaks my app - will open another issue).
